src/OmniSharp/Models/RunCodeActionResponse.cs

```diff
@@ -5,6 +5,6 @@ namespace OmniSharp.Models
     public class RunCodeActionResponse
     {
         public string Text { get; set; }
-        public IEnumerable<LinePositionSpanTextChange> Changes { get; set; }
+        public IEnumerable<ModifiedFileResponse> Changes { get; set; }
```

Comment:
@jrieken I changed the public signature here to allow for Roslyn fixes that generate new files. Is this ok with you?